webknossos-datastore/app/com/scalableminds/webknossos/datastore/controllers/ZarrStreamingController.scala

```diff
@@ -73,9 +71,7 @@ class ZarrStreamingController @Inject()(
             views.html.datastoreZarrDatasourceDir(
               "Datastore",
               "%s/%s/%s".format(organizationName, dataSetName, dataLayerName),
-              mags.map { mag =>
-                (mag.toMagLiteral(), s"$dataLayerName/${mag.toMagLiteral()}")
-              }.toMap
+              mags.map(mag => mag.toMagLiteral())
```

Comment:
if I remember correctly, sth. like `mags.map(_.toMagLiteral())` would work as well, right? Just wondering, I don't really care which version should be used.

Reply:
I’d prefer the shorter version. Also the empty call parentheses can be omitted

Reply:
```suggestion
              mags.map(_.toMagLiteral)
```

Reply:
oops, CI disagrees about the parentheses… my bad, sorry!